pkg/api/object.go

```diff
@@ -214,17 +214,18 @@ func (s *Server) listObjects(ctx iris.Context) {
 		panic(err)
 	}
 
-	ctx.Header("Content-Type", "text/vnd.yaml")
-	ctx.Write(buff)
+	w.Header().Set("Content-Type", "text/vnd.yaml")
+
+	w.Write(buff)
 }
 
-func (s *Server) getStatusObject(ctx iris.Context) {
-	name := ctx.Params().Get("name")
+func (s *Server) getStatusObject(w http.ResponseWriter, r *http.Request) {
+	name := r.URL.Query().Get("name")
```

Comment:
Actually, the object's name is in the RESTful path's `name` section.  Using `r.URL.Queyr()` can **only** get the parameters from URL after the `?` and separated by `&` token. [1] It you test the `get/update` API locally, you will get a `404` return. 

**This problem exists in every RESTful APIs which get `name` or another parameter from the URL path via a placeholder.** Plz fix them. 

## Reference
[1] https://pkg.go.dev/net/url#URL.Query